pyhocon/converter.py

```diff
@@ -276,3 +291,46 @@ def __escape_match(cls, match):
     @classmethod
     def __escape_string(cls, string):
         return re.sub(r'[\x00-\x1F"\\]', cls.__escape_match, string)
+
+    @classmethod
+    def _is_timedelta_like(cls, config):
+        return isinstance(config, timedelta) or relativedelta is not None and isinstance(config, relativedelta)
+
+    @classmethod
+    def _timedelta_to_str(cls, config):
+        if isinstance(config, relativedelta):
+            time_delta = cls._relative_delta_to_timedelta(config)
+        else:
+            time_delta = config
+        return str(int(time_delta.total_seconds() * 1000))
+
+    @classmethod
+    def _timedelta_to_hocon(cls, config):
+        """
+        :type config: timedelta
+        """
+        if relativedelta is not None and isinstance(config, relativedelta):
+            if config.hours > 0:
+                return str(config.hours) + ' hours'
```

Comment:
Pyhocon supports multiple input formats for timedelta, like:
```
1s
1 s
1 second
1 seconds
1 sec
```

But output format is fixed,and thus some applications expecting format other than this one will fail while parsing output config.

For example, Spark config values could contain time units like `150ms`, but converter will save it as `150000 microseconds` (there is just no case for milliseconds).

Reply:
@dolfinus 
output is specifically for hocon format. if spark config is different then it is not hocon and requires it's own output. don't know if it's in the scope of this library.

Reply:
I don't see [here](https://github.com/lightbend/config/blob/main/HOCON.md#duration-format) any specification about default unit format - all the options are described as supported by a parser. There is no preferred unit format, so why does implementation have one without any capability to change it whenever user wants?

Spark config is just the simple `.properties` file which is successfully read by any HOCON library implementation. But the output file saved by the current package cannot be changed at all - no class or instance options for setting up some preferred format.

Reply:
@dolfinus 
"so why does implementation have one without any capability to change it whenever user wants?" - because that is the implementation I submitted.

"Spark config is just the simple .properties" - so it's not hocon.

"file which is successfully read by any HOCON library implementation" - I do believe you are free to contribute code which enhances this library to your additional requirements. That's what I did.

ps.
From my perspective, you are coming off a bit demanding, considering this is open-source and based on free contributions from people who care enough to do so. less talk, more code :)